### ISSUE TITLE
RPC

### ISSUE THREAD

**Issue by Fuckxegg** [warp-zona/solana-trading-bot-v3#6]
I'm wondering if RPC responsiveness is the key to profitability?

**Comment by warp-zona** [comment]
> I'm wondering if RPC responsiveness is the key to profitability?

Yes, you should choose the RPC service that you think is the fastest, such as shyft or quicknode.

**Comment by Fuckxegg** [comment]
> > I'm wondering if RPC responsiveness is the key to profitability?
> 
> Yes, you should choose the RPC service that you think is the fastest, such as shyft or quicknode.

Do you recommend purchasing their premium services?

**Comment by warp-zona** [comment]
> > > I'm wondering if RPC responsiveness is the key to profitability?
> > 
> > 
> > Yes, you should choose the RPC service that you think is the fastest, such as shyft or quicknode.
> 
> Do you recommend purchasing their premium services?

That would be best, but I'd suggest switching between multiple free RPCs to test first, that's usually when you can see which one is best for you.

**Comment by Fuckxegg** [comment]
Okay, I got it. Hopefully we'll get a more objective profit🫡
